app/models/user.rb

```diff
@@ -72,5 +72,9 @@ def email=(value)
   def name
     "#{first_name} #{last_name}"
   end
- 
+
+  def ensure_district(district:)
```

Comment:
Couple of points here:

 - This method is mutating the state of the User instance, so it likely should end with a `!`.
 - The name is a bit unusual to me; it's not exactly ensuring that there's a district with that value there, it's adding one if if it doesn't exist.  Perhaps it should be `conditionally_add_district!` to better convey its intent?